Automattic/sensei
# Fix teacher can see all students on the Students screen

Resolves #6044

## Proposed Changes
* Add a hook to limit the students to ones that are part of the teacher's courses.

## Testing Instructions


1. Make sure you have a few student users.
1. Make sure you have a user with the teacher role.
1. Login with the teacher and go to Sense LMS -> Students.
1. Make sure you see no students.
1. Create a course and assign it to the teacher.
1. Have a student take that course.
1. Login with the teacher and go to Sense LMS -> Students.
1. Make sure you see only the student that has taken the teacher's course.
2. Repeat the steps above when HPPS is enabled.

## Pre-Merge Checklist
